app/views/mailers/notify_mailer/tag_moderator_confirmation_email.text.erb

```diff
@@ -15,9 +15,12 @@ To aid in communication, we’ve added you to the following <%= community_name %
 - Tag Moderators <%= app_url("/connect/tag-moderators") %>
 - #<%= @tag %>
 
-You can use these channels to meet other tag moderators across the site and co-moderators within your tag while coordinating with the <%= community_name %> team. When in doubt about how to exercise your new privileges, please ask! If these Connect channels become too distracting, you can mute or leave any channel as you wish.
+You can use these channels to meet other tag moderators across the site and co-moderators within your tag while coordinating with the <%= community_name %> team.
```

Comment:
These lines were too long and as this is the text email, they would be illegible. `erblint` complained about line length